src/main/resources/messages/messages_pl.yml

```diff
@@ -91,11 +91,11 @@ change_password_expired: 'Nie zmienisz już hasła przy użyciu tej komendy.'
 email_cooldown_error: '&cE-mail został wysłany, musisz poczekać %time przed wysłaniem następnego.'
 
 # Captcha
-usage_captcha: '&cWpisz: /captcha <theCaptcha>'
-wrong_captcha: '&cZły kod, proszę wpisać: /captcha THE_CAPTCHA'
+usage_captcha: '&cWpisz: /captcha <kod_captcha_z_chatu>'
```

Comment:
No, you don't have to traslate THE_CAPTCHA and <theCaptcha>, they are placehodlers